vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -3,29 +3,24 @@ sidebarDepth: 2
 ---
 # Export and Publish a Bundle
 
-## Overview
-Use the bundler command to export a bundle of components from an existing Entando Application. An Entando Bundle can be used 
-- For the initial install of Entando components into an Entando Application
-- To migrate Entando components from one environment to another (e.g. Dev to QA)
-- To provide a template for building a new Entando Application
-- As the skeleton of an Entando solution
+The ent bundler command can export a bundle from an existing Entando Application to be utilized in the following ways: 
+- For the initial install of components into an Entando Application
+- To migrate bundles from one environment to another (e.g. Dev to QA)
+- To provide a template for building a new application
```

Comment:
this block is kind of unnecessarily wordy; can be simplified, especially because i think the command exports a bundle by definition, not optionally.
"The ent bundler command exports a bundle from an existing Entando Application to: 
 - Perform the initial install of components into an Entando Application
 - Migrate bundles from one environment to another (e.g. Dev to QA)
 - Provide a template for building a new application"